pyproject.toml

```diff
@@ -29,6 +29,7 @@ typing-extensions = "^4.8.0"
 python-dateutil = "^2.8.2"
 tiktoken = ">=0.5.1"
 nltk = "^3.8.1"
+jsonformer = "0.12.0"
```

Comment:
Do we know what jsonformer's versioning practices are?  Can we safely allow patch updates?

Reply:
I'm not convinced we can. They're not particularly active and don't seem to be tagging releases.

Reply:
Fair enough, we can revisit if we get a custom request for it.  I think @zsimjee even mentioned potentially adopting the repo.

Reply:
yeah should be fine to lock down, it's been months since the last update